sitback-furniture/src/components/category-card/CategoryCard.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Button from '../button/Button';
+import styles from "./CategoryCard.module.scss"
+import PropTypes from "prop-types"
+import { Link } from 'react-router-dom';
+import { SHOP_NOW } from '../../constants/AppConstants';
+import { defaultImageProvider } from '../../utils/defaultImageProvider';
+
+const CategoryCard = (props) => {
+    const { category } = props;
+    return (
+        <div className={styles.categoryCardWrapper}>
+            <img src={category.photo} alt="category" onError={defaultImageProvider} />
+            <h2 className={styles.categoryTitle}>{category.category}</h2>
+            <p className={styles.categoryDescription}>{category.description}</p>
+            {/* Link tag to navigate to shopping page based on the categories choosed  */}
+            <Link to={`/categories/${category.id}`} className="btn btn-primary"><Button label={SHOP_NOW} /></Link>
```

Comment:
could you let me know if /categories can be moved to constants file and reused

Reply:
Moved the route to constants